target/linux/ath79/dts/qca9531_kuwfi_cpe2g-9531.dts

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	model = "KuWfi CPE2G-9531";
+	compatible = "qca,qca9533";
+
+	chosen {
+		bootargs = "console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd";
```

Comment:
Is this necessary as it should be default?

Reply:
to be sure of future "default" values

Reply:
Well, this won't change so please drop it as it is the default and there is no need for it.